src/NzbDrone.Core/Localization/Core/en.json

```
@@ -929,6 +929,7 @@
   "RemotePathMappingRemotePathHelpText": "Root path to the directory that the Download Client accesses",
   "RemotePathMappingWrongOSPathHealthCheckMessage": "Remote download client {0} places downloads in {1} but this is not a valid {2} path. Review your remote path mappings and download client settings.",
   "RemotePathMappings": "Remote Path Mappings",
+  "RemotePathMappingsInfo": "Remote Path Mappings are very rarely required, it is better to match your download path client to your local paths.",
```

```suggestion
  "RemotePathMappingsInfo": "Remote Path Mappings are very rarely required. If you have {app} in Docker and the Download Client in non-Docker (or vice versa) or have the programs on different servers then you may need a remote path map.",
```
updte translate implementation accordingly of course

source - https://wiki.servarr.com/sonarr/troubleshooting#remote-path-mapping

---

maybe better to link to wiki instead of a long info box? open to others thoughts also!